app/models/manageiq/providers/container_manager.rb

```diff
@@ -4,15 +4,16 @@ class ContainerManager < BaseManager
     include SupportsFeatureMixin
 
     has_many :container_nodes, :foreign_key => :ems_id, :dependent => :destroy
-    has_many :container_groups, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :container_groups, -> { active }, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_services, :foreign_key => :ems_id, :dependent => :destroy
     has_many :container_replicators, :foreign_key => :ems_id, :dependent => :destroy
-    has_many :containers, :foreign_key => :ems_id
-    has_many :container_projects, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :containers, -> { active }, :foreign_key => :ems_id
+    has_many :container_definitions, -> { active }, :foreign_key => :ems_id
```

Comment:
@kbrock yes that is defined on all_xy relations, that include also soft-deleted items